CMake/FollyCompilerUnix.cmake

```diff
@@ -26,6 +24,20 @@ function(apply_folly_compile_options_to_target THETARGET)
       -Wunused-label
       -Wunused-result
       -Wnon-virtual-dtor
-      ${FOLLY_CXX_FLAGS}
-  )
+      ${FOLLY_CXX_FLAGS})
+
+  if (APPLE)
+    target_compile_options(${THETARGET}
+      PRIVATE
+        -Wno-nullability-completeness # See https://github.com/facebook/folly/issues/740
+        -Wno-inconsistent-missing-override # See https://github.com/facebook/folly/pull/745
+        -Wno-format # See https://github.com/facebook/folly/issues/757
```

Comment:
I think we should get the string format fixes in from pull request 765 instead.
If we land those, can we drop this flag?

Reply:
Yep, we can drop the `-Wno-format` if #765 PR goes in first. I'll operate on the assumption that 765 will go in first and just remove it on this branch.